.github/workflows/linux_neovim.yml

```diff
@@ -31,21 +31,24 @@ jobs:
           mkdir -p ~/nvim/bin
           curl -L https://github.com/neovim/neovim/releases/download/${{matrix.neovim_version}}/nvim.appimage -o ~/nvim/bin/nvim
           chmod u+x ~/nvim/bin/nvim
-      - name: Download language servers
+      - name: Install Go for gopls
+        uses: actions/setup-go@v2
+        with:
+          go-version: '^1.16.1'
+      - name: Install gopls
         shell: bash
         run: |
-          mkdir -p ~/langservers
-          curl -L https://github.com/rust-analyzer/rust-analyzer/releases/download/2020-12-21/rust-analyzer-linux -o ~/langservers/rust-analyzer
-          chmod u+x ~/langservers/rust-analyzer
-          ~/langservers/rust-analyzer --version
+          go get golang.org/x/tools/gopls@latest
```

Comment:
I thought using the latest stable version is better. But if you prefer fixed versions, yes we can pin the versions. I'll try to specify them with environment variables.

Reply:
@prabirshrestha I pinned Go and gopls version through `VIM_LSP_GO_VERSION` and `VIM_LSP_GOPLS_VERSION` at https://github.com/prabirshrestha/vim-lsp/pull/1108/commits/eac78f904ccf7e4dd17c008139833e81766a5dee.

Reply:
I prefer pinning since the builds are reproducible.

A simple example that could break without pinning is might be the server today shows basic hover information but in future it may show more information. similar with diagnostics they may catch more warnings.

instead of going into a rabbit hole of why the test is failing, it becomes easier to know a test fails due to a specific gopls update.

i think it is worth it in the long run though we would have to bump the version now and then.